Tests/ParseSwiftTests/ParseObjectTests.swift

```diff
@@ -221,6 +221,57 @@ class ParseObjectTests: XCTestCase { // swiftlint:disable:this type_body_length
         wait(for: [expectation2], timeout: 20.0)
     }
 
+    func testAnyEncodableHash() throws {
```

Comment:
Test currently doesn't pass as the hasher doesn't create a unique consistent hash for the same object.